components/_expanding-components/accordion-checkbox-full-width.hbs

```diff
@@ -2,46 +2,46 @@
     <div class="msds-accordion msds-accordion--list-content msds-accordion--full-width msds-bg-gray-gradient-2">
         <div class="row">
```

Comment:
seems odd we have rows and cols in  the middle of our component?

Reply:
same as before, the msds-accordion is the main component and the row contains accordions, this code has been reviewed and approved

Reply:
im still not a fan :)

Reply:
we don't have time to redesign a component because your not a fan :P Especially when this code has already passed a CR from 2 or 3 devs in the team.

Stephan wants to do a mild refactor today and we will see what comes from that but the conversation that we are having is out of the scope for the task I had. My task was to implement unchecking radio buttons